src/phylum/ci/ci_base.py

```diff
@@ -75,7 +64,27 @@ def __init__(self, args: Namespace) -> None:
             os.environ[ENVVAR_NAME_TOKEN] = args.token
         self._args.token = token
 
-        self.ensure_project_exists()
+        self._ensure_project_exists()
+
+    def _backup_project_file(self) -> None:
+        """Create a copy of the original `.phylum_project` file values, when the file exists.
+
+        This is necessary because it is possible that user-provided values for the project and
+        group are given, which causes the file to be overwritten when creating that project.
+        """
+        cmd = [str(self.cli_path), "status", "--json"]
+        try:
+            status_output = subprocess.run(cmd, check=True, capture_output=True, text=True).stdout.strip()  # noqa: S603
+        except subprocess.CalledProcessError as err:
+            msg = "Phylum status check failed"
+            raise PhylumCalledProcessError(err, msg) from err
+        self._project_settings: Dict = json.loads(status_output)
+        project_root = self._project_settings.get("root")
+        if project_root:
+            self._phylum_project_file = Path(project_root).joinpath(".phylum_project").resolve()
+        else:
+            self._phylum_project_file = Path.cwd().joinpath(".phylum_project").resolve()
+        self._project_file_already_existed = self._phylum_project_file.exists()
```

Comment:
Isn't CI usually immutable anyway? So any changes you make don't get pushed?

Reply:
There are `CINone` and `CIPreCommit` implementations available for running `phylum-ci` locally, where the changes would be noticed. A warning is emitted when a previous `.phylum_project` file is overwritten.